### PR TITLE
updated and clarified valid service group names

### PR DESCRIPTION
**Pull request by davidwrede**
- Clarified statements and examples around service group names.
- Minor formatting and copyediting on existing text.

Signed-off-by: David Wrede <dwrede@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
@davidwrede, thanks for your PR! By analyzing the annotation information on this pull request, we identified @juliandunn and @leonhartX to be potential reviewers

File: www/source/docs/run-packages-service-groups.html.md [hunk 1]
```diff
@@ -3,44 +3,58 @@ title: Run packages in a service group
...
+    hab-sup(GS): Census myapp.prod: c60bfa12-c913-4f89-b4f3-d082d1310c3d
     ...
 
-Services only join together to form a ring when a peer IP address is specified by other similar services at runtime.  In a new window or external runtime format, run the same command you ran above, but this time, reference the peer IP address of the previous service specified in the supervisor output above.
+Services only join together to form a ring when a peer IP address is specified by other similar services 
+at runtime.  In a new window or external runtime format, run the same command you ran above, but this time, 
+reference the peer IP address of the previous service specified in the supervisor output above.
 
-    hab start myorigin/myapp --group myapp.prod --peer 172.17.0.2
+    hab start myorigin/myapp --group prod --peer 172.17.0.2
```
Comment (metadave) [inline www/source/docs/run-packages-service-groups.html.md]: should it be noted that the default port of `9634` is used if you don't specify one with `--peer`?

**Comment by metadave** [comment]
:+1:

File: www/source/docs/run-packages-service-groups.html.md (hunk 1 above)
Comment (davidwrede) [inline www/source/docs/run-packages-service-groups.html.md]: Good feedback. I added a note in the second commit.

**Pull request merged**